docs/examples/plot_interval_transposition_error.py

```diff
@@ -0,0 +1,157 @@
+"""
+Modeling with interval averages
```

Comment:
most, but not all, of the other examples are in Title Case. Fine to leave it as is and kick the standards down the road.